core/modules/image/image.field.inc

```diff
@@ -161,7 +162,15 @@ function image_field_instance_settings_form($field, $instance) {
     '#description' => t('The title attribute is used as a tooltip when the mouse hovers over the image.'),
     '#weight' => 11,
   );
-
+  $form['orientate'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Enable image re-orientation'),
+    '#description' => t('If EXIF data indicates the need, the image will be rotated appropriately before being saved. <br>
+    Note that as with resizing, rotating images on upload will cause the loss of EXIF data in the image.'),
```

Comment:
Remove `<br>` and whitespace from within `t()`, use separate `t()` calls if needed.

Reply:
Done.

Reply:
Looks like the `<br>` and whitespace is actually still here?

Reply:
Ooops! Done now.